composer/algorithms/stochastic_depth/README.md

```diff
@@ -0,0 +1,92 @@
+# 🎰 Stochastic Depth (Sample-wise)
```

Comment:
```suggestion
# 🎰 Stochastic Depth (Sample)
```

shorten this so no multi-line in the menu -- should do the same for Block

Reply:
Done